### DISCUSSION TITLE
How do you know which packages needs update?

### DISCUSSION THREAD

**Discussion by nova77** [pkgxdev/pkgx#567] [Help]
According to the docs it's possible to update specific packages with the `--sync` command, eg. `tea --sync +deno.land`.
However, I could not find any place (besides tea/gui) where I can find a list of packages that can be updated.

In addition to this, is there a way to update all packages at once? Something like `tea --sync all`.

**Comment by mxcl** [comment]
This is only currently available in the gui.

Probably `tea --sync` outside a dev-env should update all tho.

**Reply by nova77** [reply]
Thanks Max! Yeah, `tea --sync` feels like the best place. It would also be great to have a flag which shows what's available to upgrade, with old/new versions.

**Comment by mittalyashu** [comment]
It would be nice to have an interactive mode to allow selective packages to be upgrade.

For example, in `pnpm` CLI there is an option to select specific packages.

![image](https://github.com/teaxyz/cli/assets/29014463/6cb22ab7-edcf-46ab-ad28-e311864d359c)
